src/main/java/cli/scrape.java

```diff
@@ -13,28 +15,61 @@
    @Todo: Add annotation for parameter. Fix the method to parse
    @Help: Add annotations, comments to code
 */
-@CommandLine.Command(name = "scrape")
-public class scrape {
+@CommandLine.Command(
+        name = "scrape", synopsisSubcommandLabel = "(catalog | section | sections | school)",
```

Comment:
Do you think it makes sense to have both `section` and `sections` ? We might want to just get rid of `section` and make `sections` able to do singular sections without batching. Like if I do `schedge scrape sections --term 1204 --registrationNumber 120442` it should have the same behavior as `schedge scrape sections --term 1204 --registrationNumber 120442` has right now. What do you think?

Reply:
Yeah sounds good. I can make that happen. It will make the code blocks a bit shorter too.

Reply:
Yeah this sounds good. I can make that happen. It will prob make the code shorter